calc/src/mechanics/gen8.ts

```diff
@@ -784,8 +788,8 @@ function calculateGen8(
     atMods.push(0x2000);
     description.attackerAbility = attacker.ability;
   } else if (
-    attacker.hasAbility('Gorilla Tactics') && 
-    (['Gmax', 'Dynamax'].indexOf(defender.name) !== -1)
+    attacker.hasAbility('Gorilla Tactics') &&
+    ['Gmax', 'Dynamax'].indexOf(defender.name) !== -1
```

Comment:
Can we not just check `defender.isMax` now?

Reply:
That can be done in my other PR (#371), but that targets master. What's the state of the Dynamax branch going to be after this merge? Will it be merged into master?
I could also just fix Gorilla Tactics here (one-line change) and close #371.

Reply:
Yes after I test these changes with the dynamax branch this will be merged